app/experimenter/nimbus-ui/src/lib/dateUtils.test.ts

```diff
@@ -24,6 +24,12 @@ describe("humanDate", () => {
   it("with year set to 'past', should produce the date, month, and year", () => {
     expect(humanDate(FAKE_ISO_DATE, "past")).toEqual("Dec 25, 2020");
   });
+
+  it("normalizes time and timezone", () => {
+    expect(humanDate("2021-06-21T10:00:00Z")).toEqual("Jun 21");
+    expect(humanDate("2021-06-21T00:00:00Z")).toEqual("Jun 21");
+    expect(humanDate("2021-06-21")).toEqual("Jun 21");
```

Comment:
Hmm, just out of curiosity I commented out the changes to `dateUtils.ts` to see this test fail.

First case passes. Second case fails as I'd expect.

This third case doesn't fail, at least not right at this moment. That makes sense, because the issue is tied to client-side timezone. Not sure there's really a way to test that. so 🤷

Reply:
Interesting - the third case fails for me. But yeah I'm 4 hours ahead of you?
![Screen Shot 2021-06-22 at 3 47 23 PM](https://user-images.githubusercontent.com/6392049/122982396-32160080-d371-11eb-808c-77c935632e00.png)

Reply:
> But yeah I'm 4 hours ahead of you?

Yup, I'd guess that's why. I hate timezones.